meta.go

```diff
@@ -180,20 +180,20 @@ func (r Relation) IsImplicit() bool {
 // only applies to JSON because Meta has a custom
 // YAML marshaller.
 type Meta struct {
```

Comment:
I'd be tempted to add a TODO (maybe a bug in lp) about the difference in yaml/bson tag names